packages/expo-cli/src/commands/start.js

```diff
@@ -92,6 +99,48 @@ async function action(projectDir, options) {
   log.nested(chalk.green('Logs for your project will appear below. Press Ctrl+C to exit.'));
 }
 
+async function validateDependenciesVersions(projectDir, exp, pkg) {
+  if (!Versions.gteSdkVersion(exp, '33.0.0')) {
+    return;
+  }
+
+  const bundledNativeModules = await JsonFile.readAsync(
```

Comment:
Shouldn't we check if this file exists? It was first added in `expo@33.0.0` but I guess the `if` above only checks the `sdkVersion` in `app.json`, right? I can imagine someone may have `sdkVersion: "33.0.0"` but `expo@32.0.0` installed 😅